scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -4124,6 +4124,16 @@ def run(self):
                 self.__report_container_metrics,
             )
         )
+
+        server = self._global_config.scalyr_server
+        server = server.replace("https://agent.", "https://www.")
+        server = server.replace("https://log.", "https://www.")
+        global_log.info(
+            "View data from this agent at: %s/events?filter=$serverHost%%3D%%27%s%%27"
```

Comment:
I actually would like a URL here that pulls up just the `agent.log`.  A customer is going to see this when they execute `kubectl logs <agent-pod>`.  If they are doing that, I believe they are looking for the logs from this particular application process.. so it would be best to give them a way to do that.

You could do this by adding in a term "$logfile = '/var/log/scalyr-agent-2/agent.log'" to the filter term in the URL.  So, if we are making this common code with `status -v`, you'll have to add in an optional parameter to restrict the logfile to a certain file.

I'm also fine if you want to include both URLs (one for just the agent log and one for all logs output from this server host) but we'd have to make it very clear which is which.  Though, to be honest, looking at all data being gathered by this agent might just to more than they really want here.